src/components/button/styled.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { Button } from "@rmwc/button"
+import { CircularProgress } from "@rmwc/circular-progress"
+// import "@rmwc/circular-progress/circular-progress.css"
+import { getColor } from "../../theme/utils"
+import { ButtonType } from "./button"
+
+export const StyledButton = styled(({ label, ...otherProps }) => (
+  <Button label={label} {...otherProps} />
+))`
+  ${props => {
+    if (props.disabled) {
```

Comment:
You can use [propOrElse](https://github.com/netdata/netdata-ui/blob/master/src/components/icon/icon.stories.tsx#L46-L59)

Reply:
Ok. I can try that but because of the arrow function here let me try this in one next iteration